Doesn't seem to work with Hyper 1.0.0 on Windows

And this is what the console shows:

![drag](https://cloud.githubusercontent.com/assets/2685357/21163396/811d2452-c19b-11e6-957b-8352f81bfdd6.PNG)

Where does the `el` object comes from?

```
bundle.js:1 Error: Cannot find module 'C:\Users\Benjamin\.hyper_plugins\node_modules\hyperterm-tabs'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (C:\Users\Benjamin\AppData\Local\hyper\app-1.2.1\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at require (internal/module.js:20:19)
    at de.plugins.concat.map.ge (file:///C:/Users/Benjamin/AppData/Local/hyper/app-1.2.1/resources/app/dist/bundle.js:1:8503)
    at Array.map (native)
    at ie (file:///C:/Users/Benjamin/AppData/Local/hyper/app-1.2.1/resources/app/dist/bundle.js:1:8457)
    at Object.<anonymous> (file:///C:/Users/Benjamin/AppData/Local/hyper/app-1.2.1/resources/app/dist/bundle.js:1:9992)
    at __webpack_require__ (file:///C:/Users/Benjamin/AppData/Local/hyper/app-1.2.1/resources/app/dist/bundle.js:1:119)
    at Object.<anonymous> (file:///C:/Users/Benjamin/AppData/Local/hyper/app-1.2.1/resources/app/dist/bundle.js:45:375898)
```

--- a/index.js
+++ b/index.js
@@ -255,7 +255,7 @@ exports.decorateTerm = (Term, { React }) => {
                 }
 
                 if (this.props.isTermActive) {
-                    const doc = this.el.term.document_;
+                    const doc = this.el.props.term.document_;
 
                     this.keys = new Mousetrap(doc);
 

